File: Python/card.py
from payment import Payment

class Card(Payment):

    number = str
    date   = str
    cvv    = str

    def __init__(self, id,number,date,cvv):
        super().__init__(id)
        self.number    = number
        self.date      = date
        self.cvv       = cvv

File: Java/Card.java
package Java;

import java.sql.Date;

class Card extends Payment {
    String number;
    Date date;
    String cvv;

    public Card(Integer id, String number, Date date, String cvv) {
        super(id);
        this.number = number;
        this.date = date;
        this.cvv = cvv;
    }

}
